### ISSUE TITLE
No fixtures created when running with xdist

### ISSUE THREAD

**Issue by SBoudrias** [pytest-dev/pytest-django#285]
I have a class based test architecture. When I'm running `py.test some/test/file.py`, I get green tests. But once I run passing `-n 2` with xdist, then for some reason my fixtures are not created.

```py
class BaseTest(object):
    fixtures = ['fixture-file.json']

class Test(BaseTest, TestCase):
    def test_something():
        # At this point no fixtures is loaded if we're in an xdist process
```

Any idea if that's a bug or if something I'm doing might be confusing the system?

(I know the architecture described here is not ideal. But I'm working on a fairly large legacy code base.)

**Comment by pelme** [comment]
Thanks for the report!

Are you using the latest version (2.9.1)? 2.9.0 introduced a regression with regard to mixins like your BaseTest class - this issue may be related.

(We are trying to as compatible with existing Django tests we possibly can, and the example you show here should just work.)

**Comment by SBoudrias** [comment]
Yes, this is happening in 2.9.1. I also tried to with 2.8.0, and I had the same issue.

Any ideas as to why this occurs only when tests are running in parallel?

For what it worths, it also looks like pytest marks set on base classes/mixins are not inherited by the childs when running in parallel.

**Comment by pelme** [comment]
Can you reproduce this outside of your big project with a small self-contained example?
